### DISCUSSION TITLE
Capacitor Browser - Session Cookie not working?

### DISCUSSION THREAD

**Discussion by maquejp** [ionic-team/capacitor#5632] [Q&A]
Hello,
I am trying to open an authentication site (very specific CAS) but when I enter the credentials, it considers that the session is expired.
I suspect that the session cookie is not stored correctly.
Am I right? How can I debug?
I do my test on Android Studio, Pixel 5 API 30
Thank you

**Comment by w-mazed** [comment]
Hi @maquejp,

It could be a server side issue, you have to give more details to expose your case.

Best regards.
